docs/README.md

```diff
@@ -1,12 +1,13 @@
 # LogDNA Agent
 
-[![Rustc Version 1.42+]][rustc] [Join us on LogDNA's Public Slack]
+[![Rustc Version 1.46+]][rustc]
+[Join us at the LogDNA community forum]: https://community.logdna.com
```

Comment:
I'm not sure why the rendered Markdown for me shows `https://community.logdna.com` separately, rather than just a link to the text, `Join us at the LogDNA community forum`. Also should `community forum` be lower case or capitalized?

Reply:
Ah, yes, Jorge had the same question. I actually want the URL to show.. I want people to learn it. ;-)

And yeah, lower-case is ok. We could go either way... let me know if you feel strongly about it @alfeng6 .